### ISSUE TITLE
Goturn trained on ALOV only

### ISSUE THREAD

**Issue by subhadip3** [davheld/GOTURN#68]
Hi, 
I ran a Goturn(Tensorflow) implementation on ALOV dataset only. When I run it on a test set I find all the bounding boxes at the centre of the image. Does this make sense? Since among majority of the search/target(where obj is centred) pairs in ALOV there is not too much change.

**Comment by anjali1601** [comment]
@subhadip3  Did you find a solution?

**Comment by davheld** [comment]
1) My code generates synthetic image "shifts" by cropping the image in different ways, so this bias in ALOV shouldn't cause too much of a problem.  

2) Also, at test time the bounding box is typically initialized at the location of the object and then the output prediction is a relative motion, relative to the location of the box in the previous frame.  So the box at test time shouldn't necessarily go to the center of the image unless that's where it starts.
